src/demo/ShareTooltip/index.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
```

Comment:
All files should be moved under `__stories__` of ShareTooltip.  See Button stories for example of showcase demo